### DISCUSSION TITLE
Add ability to follow artist on search page

### DISCUSSION THREAD

**Discussion by lgg** [ciderapp/Cider#800] [Added Features]
For now I need to open artist's page to follow. Please add dropdown menu on artists result element on search results like on album.

![Peek 2022-04-03 12-31](https://user-images.githubusercontent.com/12018713/161421225-75b102bd-ffb4-4f0d-a52d-b0a0f6e3fba9.gif)

# Current :hover buttons

![Peek 2022-04-03 17-13](https://user-images.githubusercontent.com/12018713/161432531-7b5ac6f2-fabf-4cf4-b567-596625f22f3e.gif) <img src="https://user-images.githubusercontent.com/12018713/161432524-e0162e28-c5da-4fbd-ab99-602b25cfe517.png" width="140px"> 

# Suggestion

## add new button on :hover effect (`mouseover`)
<img src="https://user-images.githubusercontent.com/12018713/161432551-bc8b08d0-08ab-42d6-bf45-cd35bf1e643c.png" width="200px"> 

## add such-like menu to this button
<img src="https://user-images.githubusercontent.com/12018713/161432555-b95c8e60-17dd-4177-a194-50a11c14aab6.png" width="300px">

**Comment by Amaru8** [comment]
**Hi @lgg, I have some great news!** This feature is now implemented in the latest development version of **Cider**, and will be included in the next Microsoft Store update.

![Cider_800_1](https://user-images.githubusercontent.com/52407090/165586187-0f56291d-2bca-409e-9d8c-484dc14eb176.png)

![Cider_800_2](https://user-images.githubusercontent.com/52407090/165586196-1c5c4501-f541-4be3-b54d-d0eee266ac35.png)

_(Actual implementation)_
